src/main/java/com/iop/productservice/service/ComponentService.java

```diff
@@ -41,4 +41,10 @@ public ResponseEntity<Component> createComponent(ComponentRequest request) {
         }
         return new ResponseEntity<>(created, HttpStatus.CREATED);
     }
+
+    public ResponseEntity<String> deleteComponent(Long componentId) {
+        getComponent(componentId);
```

Comment:
writedown some loggers in service implementation method

Reply:
implementation done